bHY6yh-d50443/cod-bxxqqq_bHY6yh-d50443.xml

```diff
@@ -0,0 +1,396 @@
+          <lb ed="#B"/><ref xml:id="bHY6yh-d50443-Rd1e589">LEx codatis titulo de repudio<!--line jumps up--> libro III</ref><!--line jumps down--> <quote xml:id="bHY6yh-d50443-Qd1e516">dubium 
+          <lb ed="#B"/>non est / omnia que consilio ratione 
+          <lb ed="#B"/>geruntur iure merita que effectum et 
+          <lb ed="#B"/>firmitate uiti</quote>
+        </p>
+        <p xml:id="bHy6yh-d50443-d1e522">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d50443-Rd1e602">MArcialis cocus<!--line jumps up--> libro vndique suscep<!--line jumps down-->to</ref> ¶ <quote xml:id="bHY6yh-d50443-Qd1e526">pannorum uete<lb ed="#B" break="no"/>rum 
+          facile contempnitur vsus /
+          <lb ed="#B"/>¶ Non sic consilium posthume superne sce<lb ed="#B" break="no"/>num
+          ¶ Consilio Inuenum fidis me<lb ed="#B" break="no"/>libee 
+          ruinam ¶ Expectare potes 
+          <lb ed="#B"/>de re militari</quote><!--appears the scribe skipped a line with this line-->
+        </p>
+        <p xml:id="bHY6yh-d50443-d1e535">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d50443-Rd1e620">UEgetius de re<!--line jumps up--> militari libro I titulo<!--line jumps down--> 26 et fine</ref> ¶ <quote xml:id="bHY6yh-d50443-Qd1e539">Nulla 
+          <lb ed="#B"/>consilia sunt meliora quam illa 
+          <lb ed="#B"/>que ignorauerit aduersarius 
+          <lb ed="#B"/>ante quam facias ¶ Cum consilium tuum 
+          <lb ed="#B"/>cognouerit aduersarius proditum 
+          <pb ed="#B" n="86-r"/>
+          <cb ed="#B" n="a"/>
+          <lb ed="#B"/>dispositionem mutare te conuenit /</quote>
+        </p>
+        <p xml:id="bHY6yh-d50443-d1e551">
+          <lb ed="#B"/>¶ <ref xml:id="bHY6yh-d50443-Rd1e636"> </ref> <quote xml:id="bHY6yh-d50443-Qd1e553">Quod fieri debeat tractato cum 
+          <lb ed="#B"/>multis quod uero facturus sis cum 
+          <lb ed="#B"/>paucissimis vel potius ipse te<!--line jumps down-->cum</quote>
+        </p>
+        <p xml:id="bHY6yh-d50443-d1e561">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d50443-Rd1e651">UAlerius maximus libro 7 titulo 
+          <lb ed="#B"/>vafre dicta</ref> ¶ <quote xml:id="bHY6yh-d50443-Qd1e565">Hasdrubal a 
+          <lb ed="#B"/>Iulio proconsule nauali prelio 
+          <lb ed="#B"/>uictus timensque classis amis<lb ed="#B" break="no"/>se 
+          penas dari / offensam astuti<lb ed="#B" break="no"/>a 
+          auertit nam ex illa infelici 
+          <lb ed="#B"/>pugna prius quam cladis nun<lb ed="#B" break="no"/>cius 
+          domum perueniret. quendam 
+          <lb ed="#B"/>ex amicis compositum et ornatum 
+          <lb ed="#B"/>cartaginem misit qui postquam 
+          <lb ed="#B"/>ciuitatis eius curiam intrauit 
+          <lb ed="#B"/>Consulit vos inquit hasdrubal 
+          <lb ed="#B"/>cum dux romanorum magnas secum 
+          <lb ed="#B"/>maritimas trahens copias ad<lb ed="#B" break="no"/>uenerit. 
+          an cum eo conf<underline>i</underline>ligere debeat 
```

Comment:
```suggestion
          an cum eo conf<hi rend="underline">i</hi>ligere debeat 
```

Reply:
There are two files edited in this branch. 

We need to create a new branch called p3-b2-r3-redo that only has changes in the file for rubric 3. Then I can accept the PR from the redo branch and reject this branch PR